content/en/docs/contributing/release-process.md

```diff
@@ -172,99 +218,42 @@ version before it.
-
-4. Run `cmrel publish`
-
-   1. First do a dry-run, to ensure that all the staged resources are valid.
-
-       ```bash
-       cmrel publish --release-name <RELEASE_NAME>
-       ```
-
-        Where `<RELEASE_NAME>` is the unique build ID printed by the earlier `cmrel stage` command.
-        E.g. Given `gs://cert-manager-release/stage/gcb/release/v1.0.2-219b7934ac499c7818526597cf635a922bddd22e`,
-        the `RELEASE_NAME` would be `v1.0.2-219b7934ac499c7818526597cf635a922bddd22e`.
-
-        You can view the progress by clicking the Google Cloud Build URL in the output of this command.
-
-   2. Next publish the release artifacts for real.
-
-        If the last step succeeded, you can now re-run the `cmrel publish` with the `--nomock` argument
-        to actually publish the release artifacts to GitHub, `Quay.io`, [Helm Hub] etc.
-
-        ```bash
-        cmrel publish --nomock --release-name <RELEASE_NAME>
-        ```
-
-        NOTE: At this stage there will be a draft release on GitHub and a live release on [Helm Hub].
-        So you must now complete the release process quickly otherwise users of the latest release on [Helm Hub] will encounter errors,
-        because the manual CRD install URL will not be available yet.
-
-5. Publish the GitHub release
-
-   1. Visit the draft GitHub release and paste in the release notes that you generated earlier.
-
-        You will need to manually edit the content to match the style of earlier releases.
-      In particular:
-
-      * Remove package related changes
-      * Replace links to `@jetstack-bot`, in cherry-pick PRs, with links to the GitHub handle of the author of the original PR.
-
-   2. Click "publish" to make the GitHub release live.
-      This will create a Git tag automatically.
-
-6. Finally post a link to the release tag to all cert-manager channels on Slack.
-
-    E.g. `https://github.com/jetstack/cert-manager/releases/tag/v1.0.2 :tada:`
-
-## Release Notes
-
-1. Download and install the latest version of [Kubernetes Release Notes Generator](https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md).
-
-2. Run as follows, substituting the current and last versions where appropriate:
+1. Run as follows, substituting the current and last versions where appropriate:
 
     ```bash
     export GITHUB_TOKEN=*your-token*
     export RELEASE_VERSION=1.0.0
     export BRANCH=release-1.0
     export END_REV=release-1.0
     export START_REV=v0.16.1
-    $GOPATH/bin/release-notes --github-repo cert-manager --github-org jetstack --required-author "jetstack-bot" --output release-notes.md
+    krel release-notes --github-repo cert-manager --github-org jetstack --required-author "jetstack-bot" --output release-notes.md
     ```
 
-    NOTE: The GitHub token needs read-only permission to the cert-manager repository. 
+    NOTE: The GitHub token needs read-only permission to the cert-manager repository.
```

Comment:
Nit: In the Prerequisites section it says that the token 'does not need any privileges'. I guess since the cert-manager repository is public, no-privileges and read-only permissions are equivalent? But maybe it's worth to use the same wording in both places.